src/BrandwatchReactAuth.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import jwtDecode from 'jwt-decode';
+import TokenStore from 'viziaauth';
+
+export default class BrandwatchReactAuth extends Component {
+  getChildContext() {
+    return {
+      brandwatchAuthLogout: this.handleLogout,
```

Comment:
Or just to get access to the token in a component, my react is a lil rusty, but smth like:
```js
class MyComponent extends React.Component {
  handleClick(data) {
    return dispatch => this.context.getToken().then(token =>
      dispatch(someApiAction(token, data))
    )
  }
}
```